test/typescript/index.tsx

```diff
@@ -0,0 +1,92 @@
+import * as ReactDOM from 'react-dom';
```

Comment:
Why do we need this file?

Reply:
I added it for the [`ReactDOM.render`](https://github.com/ai/storeon/pull/24/files#diff-2bc471cb1a492c249aaec98d1e7a4735R87). Anyway, this package exists only for typescript as a part of `@types/react` definition; I didn't install the real package.

Reply:
Why we need `test/typescript/index.tsx`?

Reply:
It is a typing test that contains real code but does not execute it, just checks the correctness of types. This kind of tests are used to avoid type inconsistensy in typescript definitions.

Unfortunately, I'm not familiar with Preact, so I can write tests only for React. I renamed `index.tsx` to `react.tsx` to make it clearer.